WebRoot/templates/mail/Message.template

```diff
@@ -346,7 +346,7 @@
 </template>
 
 <template id='mail.Message#AllAttachments'>
-	<table role="presentation" class="ZmMailAttachmentsAllTable">
+	<table role="presentation" class="ZmAttachmentsAllTable">
```

Comment:
It's just for semantic reason. Because this classes are no longer specific to mail attachments only. It is also applicable to task and appointment attachments.